.gitignore

```diff
@@ -61,3 +61,6 @@ _build/
 # cypress artifacts
 cypress/videos
 cypress/screenshots
+
+#asdf vm
```

Comment:
asdf vm??

Reply:
When I set this up locally without the docker context I used [asdf version manager](https://asdf-vm.com/). It can be used for Python, JS, Ruby, Elixir... To avoid committing the `.tool-versions` file I added it to the `.gitignore`

Reply:
Would it be worth including the url you provided above? It does look like a random string atm 😄

```suggestion

# asdf-vm (https://asdf-vm.com/)
```